src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -245,4 +245,16 @@ public static void forEachChild(DetailAST root, int type, Consumer<DetailAST> ac
         }
     }
 
+    /**
+     * Determines if two ASTs are on the same line.
+     *
+     * @param ast1   the first expression
+     * @param ast2   the second expression
+     *
+     * @return true if they are, false otherwise
+     */
+    public static boolean areOnSameLine(DetailAST ast1, DetailAST ast2) {
```

Comment:
Since it is a boolean result, shouldn't the method be named `isOnSameLine`?

Reply:
changed

Reply:
@rnveach , can you explain why `is` should be used ? I thought as we have two arguments, so plural from `are` should be used. 
`is` might good if we had code like `ast.isOnSameLine(ast1)`

Reply:
@romani Truthfully I never really paid attention to the number of parameters. I always try to name methods "is" (present or past) based on what they are doing.

I fine we keeping it as `are`.

Reply:
I always try to name all to way to make code read as English text.
To honor almost all rules. In this context we use context less function  (static method). So as we have two arguments, so there plural form should be used, and looks good, so we have to use it.

I am voting to use `are`.

Let's keep this PR hanging till we agree on naming convention for such cases.

Please vote.

Reply:
I am fine with both options, but `are` looks slightly better for me.

Reply:
Please make the change back then.

Reply:
ok done